.github/workflows/deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy
+
+on:
```

Comment:
We should merge both the workflow files so that the library only gets uploaded if the tests pass. Here's example pseudocode for the relevant jobs:
```yaml
bintray-upload:
  if: github.ref == 'refs/heads/master'
  needs: [ tests ]
```